src/features/swap/helpers/utils.ts

```diff
@@ -24,6 +24,9 @@ function asDecimal(amount: number | bigint, decimals: number): number {
 
 export const TWAP_FALLBACK_HANDLER = '0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5'
 
+// https://github.com/cowprotocol/composable-cow/blob/main/networks.json
+export const TWAP_FALLBACK_HANDLER_NETWORKS = ['1', '100', '11155111', '42161']
```

Comment:
Nit: We could also use a mapping chainId -> address here. But maybe we can change it if there are ever multiple addresses.